tests/unit/experiment/test_experiment.py

```diff
@@ -0,0 +1,47 @@
+from flambe.experiment import Experiment
+from flambe.runnable import RemoteEnvironment
+
+import mock
+import pytest
+
+
+@pytest.fixture
+def get_experiment():
+    def wrapped(**kwargs):
```

Comment:
Why this kwargs parsing is necessary at test level? Just create an Experiment directly

Reply:
I expect the `test_experiment` file to grow soon. Adding a fixture at this point is good practice so that future tests can use it!

Reply:
Yes but why the kwargs?

Reply:
This is for the user of the fixture to add the attributes that he wants (all other will receive dummy values). The `**kwargs` avoids copying all Experiment parameters.